### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Eclipse Azure Device login_Screen Reader: When focus is inside the error details section, Narrator is reading as ‘Editing’

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3323]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #NVDA; #Narrator

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start Narrator/NVDA and Open Java EE - Eclipse application 
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Tab till cancel button and press enter
6) Verify When focus is inside the error details section, Narrator is reading as ‘Editing’

**Actual Result:** 
When focus is inside the error details section, Narrator is reading as ‘Editing’
NVDA is reading as “Edit Multiline” but reading the error message

**Expected Result:** 
When focus is on the ‘Error Details’ Section narrator should read the error message as ‘Device login failed or was cancelled’ but should not read as Editing
NVDA should not read as edit but should only read the message

**User Impact:** 
Screen Reader users will not able to understand the error if screen reader is reading as just editing 

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_8_Narrator is reading error details as editing.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451189/Bug_8_Narrator.is.reading.error.details.as.editing.pptx)
![Bug_8_NVDA should not read as 'Edit multi line but should only read the error message](https://user-images.githubusercontent.com/41109135/62204423-bd1b3580-b3aa-11e9-9df4-238cf4776198.png)

**Comment by Flanker32** [comment]
We use Text component to display error message, so that users could use `tab` to find the component and read it by screen reader. We could replace it with Label so that Screen Reader will not read component name but just the error message, but this will make users unable to reach error message by `tab`

**Comment by Flanker32** [comment]
- Importance of experience: Medium,users may made some mistake like close the device login window by mistake and error dialog will show at this time.
- Quality of experience: Low, users may get a bit confused but screen reader will read the error message as well, besides, users could know this is an error popup by title `Sign in error`, and the body refers to error message.
- Alternate solutions: We could change the component to Label but users couldn't reach it by `tab`.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550839

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749744
